x-pack/plugins/security_solution/server/lib/detection_engine/rules/bulk_action_edit.test.ts

```diff
@@ -80,11 +80,14 @@ describe('bulk_action_edit', () => {
         expect(editedRule.params).toHaveProperty('index', ['initial-index-*', 'my-index-*']);
       });
       test('should remove index pattern from rule', () => {
-        const editedRule = applyBulkActionEditToRule(ruleMock as RuleAlertType, {
-          type: BulkActionEditType.delete_index_patterns,
-          value: ['initial-index-*'],
-        });
-        expect(editedRule.params).toHaveProperty('index', []);
+        const editedRule = applyBulkActionEditToRule(
+          { params: { index: ['initial-index-*', 'index-2-*'] } } as RuleAlertType,
```

Comment:
Using `as` is not very safe. I think we need to have functions for creating stub values of `RuleAlertType`, but that's out of the scope of this PR. So just a thought.

Reply:
I think this approach can work for small unit tests that test specific part(like one field), you can see with one glimpse what the payload is getting consumed by tested function.
That reduces maintenance that stub generator may require.

 But I can also see the benefit of it as well, if it's flexible and easy configurable.